tests/visualize/test_viz_tree.py

```diff
@@ -14,6 +14,7 @@
 IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning:")
```

Comment:
Strictly not part of this PR -  part of #27. - ignoring this warning, as we are not heavily dependent on it.